pkg/api/brickutils.go

```diff
@@ -1,14 +1,13 @@
-package volumecommands
+package api
 
 import (
 	"fmt"
 
-	"github.com/gluster/glusterd2/pkg/api"
-
 	"github.com/pborman/uuid"
 )
 
-func nodesFromVolumeCreateReq(req *api.VolCreateReq) ([]uuid.UUID, error) {
+// NodesFromVolumeCreateReq extracts the list of nodes from Volume Create request
+func NodesFromVolumeCreateReq(req *VolCreateReq) ([]uuid.UUID, error) {
```

Comment:
You can make this as a method of VolCreateReq structure and then use it as `req.Nodes()`